internal/connector/connector.go

```diff
@@ -136,11 +138,18 @@ func Run(ctx context.Context, options Options) error {
 	ctx, cancel := context.WithCancel(ctx)
 	defer cancel()
 
-	client := &api.Client{
-		Name:      "connector",
-		Version:   internal.Version,
-		URL:       u.String(),
-		AccessKey: accessKey,
+	promRegistry := metrics.NewRegistry(internal.FullVersion())
+	responseDuration := prometheus.NewHistogramVec(prometheus.HistogramOpts{
+		Namespace: "serverhttp",
+		Name:      "response_duration_seconds",
+		Help:      "A histogram of duration, in seconds, handling server HTTP responses",
```

Comment:
I guess this needs to be changed  to something like:
```suggestion
		Help:      "A histogram of duration, in seconds, of HTTP requests to the infra server",
```